Assets/Scripts/Gamestate/MatchController.cs

```diff
@@ -83,13 +83,31 @@ public void StartNextRound()
         rounds.Add(new Round(players.Select(player => player.playerManager).ToList()));
     }
 
+    public void StartNextBidding()
+    {
+        onBiddingStart?.Invoke();
+        SceneManager.LoadSceneAsync("Bidding");
+        PlayerInputManagerController.Singleton.playerInputManager.splitScreen = false;
+    }
+
     public void EndActiveRound()
     {
         onRoundEnd?.Invoke();
 
         AssignRewards();
 
-        CheckWinCondition();
+        if (!CheckWinCondition())
```

Comment:
Perhaps we should rename this method now that it returns a value?

Reply:
Yes